7kaa.desktop

```
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Name=Seven Kingdoms
+GenericName=Seven Kingdoms
+Comment=Seven Kingdoms: Ancient Adversaries
+Type=Application
+Exec=7kaa
+Icon=com.sevenkfans.SevenKingdomsAncientAdversaries
+Categories=Game;StrategyGame;
+StartupNotify=false
```

Add keywords it is helpful also
```suggestion
StartupNotify=false
Keywords=game;strategy;ancient;kingdoms;real-time;trade;diplomacy;espionage;7kaa;
```